projects/ccf-rui/src/app/shared/components/labeled-slide-toggle/labeled-slide-toggle.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, Output, EventEmitter, HostBinding } from '@angular/core';
+
+/**
+ * Generic toggle slider component
+ */
+@Component({
+  selector: 'ccf-labeled-slide-toggle',
+  templateUrl: './labeled-slide-toggle.component.html',
+  styleUrls: ['./labeled-slide-toggle.component.scss']
+})
+
+export class LabeledSlideToggleComponent {
+
+  @HostBinding('class') readonly clsName = 'ccf-labeled-slide-toggle';
```

Comment:
The following comment can be added above variable to make doc coverage happier: `/** HTML class name */`